src/app/features/countries/pages/profile/profile.component.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../../../styles.scss';
+
+
+.page-wrapper {
+    height: 130vh;
```

Comment:
modify all the styles in this page without using hardcoded heights similar to this. (hint: consider adding the following to your global styles
 `html, body {
  height: 100%;
}
`